nodejs/FatBeaconPeripheral/README.md

```diff
@@ -0,0 +1,30 @@
+This is a nodejs implementation of a Fatbeacon developed on a Raspberry Pi 3.
```

Comment:
I do, but you may be able to queue them up by looking at the commit and choosing to start a review...not sure if it will let you review your own change